luatest/utils.lua

```diff
@@ -148,4 +149,10 @@ function utils.get_fn_location(fn)
     return ('%s:%s'):format(fn_source[#fn_source], fn_details.linedefined)
 end
 
+function utils.generate_id(length, urlsafe)
+    if urlsafe == nil then urlsafe = true end
```

Comment:
Please swap this line with the line below.

Reply:
Fixed.